Content.Server/Cargo/Systems/CargoSystem.CVars.cs

```diff
@@ -0,0 +1,24 @@
+using Robust.Shared.Configuration;
+
+namespace Content.Server.Cargo.Systems
+{
+    public sealed partial class CargoSystem
+    {
+        [Dependency] private readonly IConfigurationManager _cfg = default!;
+    }
+
+    [CVarDefs]
+    public sealed class CargoCVars
+    {
+        /// <summary>
+        ///     Determines if a trade station spawns in its own FTL map.
+        /// </summary>
+        /// <remarks>
+        ///     Set this to true if and only if you are disabling the normal spawning method.
+        ///     Otherwise you get two trade stations.
+        ///     This does NOTHING to the trade station that spawns on the default map.
```

Comment:
```suggestion
        ///     This does NOTHING to the trade station that spawns on the default map.
        ///     To change that, in Resources/Prototypes/Entities/Stations/base.yml, comment out the "trade:" section.
```
And a similar comment should be left in trade.yml so it's *obvious* how to have either mode.